src/main/kotlin/stringcalculator/core/FormulaParser.kt

```diff
@@ -0,0 +1,32 @@
+package stringcalculator.core
+
+object FormulaParser {
+    private const val REGEX_FIND_CUSTOM_DELIMITER = "(?<=//)(.*?)(?=\n)"
+    private val DELIMITER_FIND_PATTERN = Regex(REGEX_FIND_CUSTOM_DELIMITER)
```

Comment:
어짜리 Regex 객체를 상수화 시킨다면, `"(?<=//)(.*?)(?=\n)"`를 상수로 둘 필요는 없을 것 같아요. 🤔

```suggestion
    private val DELIMITER_FIND_PATTERN = Regex("(?<=//)(.*?)(?=\n)")
```